schemas/system/Localization.schema

```diff
@@ -0,0 +1,7 @@
+System({
+  Symbol: { domain: 'Nomen', display: true, required: true },
+  Language: { category: 'Language', required: true, index: true },
+  Text: { domain: 'Nomen' },
```

Comment:
Shouldn't this be 'Text' domain, as 'nomen' is limited to 64 chars?

Reply:
Agree